### PR TITLE
L2-512: Hide remove hotkeys if not controller

### PR DESCRIPTION
**Pull request by lmuntanerDfinity**
# Motivation

Hide button to remove hotkeys unless user is controller.

# Changes

* Hide button to remove hotkeys unless user is controller.

# Tests

* Add test case in component test.

### CODE REVIEW

**Comment by lmuntanerDfinity** [comment]
UI of the hotkeys card when user is not controller

Screenshot 2022-05-05 at 13 44 31

**Comment by lmuntanerDfinity** [comment]
@mstrasinskis please review

File: frontend/svelte/src/lib/components/neuron-detail/NeuronHotkeysCard.svelte [hunk 1]
```diff
@@ -38,11 +38,14 @@
...
         <li>
-          <span>{hotkey}</span><button
-            class="text"
-            on:click={() => remove(hotkey)}
-            data-tid="remove-hotkey-button">x</button
-          >
+          <span>{hotkey}</span>
+          {#if isControllable}
+            <button
+              class="text"
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/components/neuron-detail/NeuronHotkeysCard.svelte:44]: Could you please add aria-label="close" here?

**Review by mstrasinskis — APPROVED** [review]
LGTM, just a tiny suggestion.

File: frontend/svelte/src/lib/components/neuron-detail/NeuronHotkeysCard.svelte (hunk 1 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/neuron-detail/NeuronHotkeysCard.svelte:44]: good idea!

**Pull request merged**